### PR TITLE
Strip biosdevname and net.ifnames flags for Debian 10

### PR DESCRIPTION
**Pull request by dustinmiller**
- [x] Changelog updated

### CODE REVIEW

**Comment by mmlb** [comment]
please add "why" info in the commit message.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,6 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) though uses different version number scheme (YY.MM.DD.NN) where NN is build number.
 
-## Unreleased
-
-### Added
- - workflow mode to allow users to execute workflows 
+## [20.04.10.00] - 2020-04-10
```
Comment (mmlb) [inline CHANGELOG.md]: Lets keep unreleased for now, while we figure out how we do release numbers in new world ;)

**Review by mmlb — CHANGES_REQUESTED** [review]
see comment

**Comment by dustinmiller** [comment]
Wasn't entirely sure what was being asked but added back in the unreleased section and moved this under it..

**Comment by mmlb** [comment]
> Wasn't entirely sure what was being asked but added back in the unreleased section and moved this under it..

Yep that was exactly it, :+1:

**Review by mmlb — APPROVED** [review]

**Pull request merged**